source/linear_recurrences.h

```diff
@@ -15,8 +15,12 @@ ll linear_recurrence(const vector<ll>& a, const vector<ll>& b, const vector<ll>&
 			m[k+i][k+j] = (m[k+i-1][k+j-1] + m[k+i-1][k+j]) % mod;
 	}
 
-	vector<ll> xx(k+h, 1);
+	vector<ll> xx(k+h);
 	for(int i = 0; i < k; i++) xx[i] = x[i];
+	if(h > 0) {
+        xx[k] = 1;
```

Comment:
A tab is missing here
```suggestion
		xx[k] = 1;
```